aiida/orm/computers.py

```diff
@@ -108,7 +95,6 @@ def full_text_info(self):
         ret_lines.append(" * UUID:           {}".format(self.uuid))
         ret_lines.append(" * Description:    {}".format(self.description))
         ret_lines.append(" * Hostname:       {}".format(self.hostname))
-        ret_lines.append(" * Enabled:        {}".format("True" if self.is_enabled() else "False"))
```

Comment:
Is there now an easy way to inquire whether the computer is disabled for the current user?

Reply:
In the API there is `get_user_enabled` and the `verdi computer list` still has the normal switches. Just the concept of globally enabled no longer applies

Reply:
ok